openc3/python/openc3/topics/limits_event_topic.py

```diff
@@ -239,12 +239,12 @@ def sync_system_thread_body(cls, scope, block_ms=None):
                     target_name = event["target_name"]
                     packet_name = event["packet_name"]
                     item_name = event["item_name"]
-                    target = telemetry[target_name]
+                    target = telemetry.get(target_name)
                     if target:
-                        packet = target[packet_name]
+                        packet = target.get(packet_name)
                         if packet:
-                            enabled = ConfigParser.handle_true_false_none(event["enabled"])
-                            persistence = event["persistence"]
+                            enabled = event.get("enabled", None)
```

Comment:
Is the handle_true_false needed? ie.  does enabled every get set to false?

Reply:
It gets set to `False` which is valid JSON that we decode. So we're always returning a boolean.